### PR TITLE
Add/list container

### PR DESCRIPTION
**Pull request by eduardogspereira**
## Contexto

Este PR integra a chamada na API para listar os links com a página `/payment-links`.

## Checklist
- [ ] Integra a chamada na API para listar os links com a página `/payment-links`.

## Issues linkadas
- [ ] Resolves [CRED-25](https://mundipagg.atlassian.net/secure/RapidBoard.jspa?rapidView=186&projectKey=CRED&modal=detail&selectedIssue=CRED-25)

## Como testar?
Rodar o storybook :)

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: daba122286661410ffbb168948b5f53deb1b68d4
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: daba122286661410ffbb168948b5f53deb1b68d4
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: daba122286661410ffbb168948b5f53deb1b68d4
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: daba122286661410ffbb168948b5f53deb1b68d4
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/cockpit/src/paymentLinks/export/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import buildQuery from '../search/query'
+import buildResult from './result'
+
+const exportData = client => (filter, exportType) => {
+  const query = buildQuery(filter)
+
+  return client.search({
```
Comment (eduardogspereira) [inline packages/cockpit/src/paymentLinks/export/index.js:7]: Duplicada a chamada

File: packages/pilot/src/hooks/useFileExporter.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+const handleXLSExportDownloadingClick = (data, filename) => {
+  const workSheetName = 'sheetJS'
+  const newWorkSheet = XLSX.utils.book_new()
+  const dataWorkSheet = XLSX.utils.aoa_to_sheet(data)
+  XLSX.utils.book_append_sheet(newWorkSheet, dataWorkSheet, workSheetName)
+
+  XLSX.writeFile(newWorkSheet, filename.concat('xls'))
+}
+
+const useFileExporter = (filePrefix) => {
```
Comment (eduardogspereira) [inline packages/pilot/src/hooks/useFileExporter.js:36]: Não da pra receber as colunas aqui e remover do exportData.
Não é melhor passar os dados de uma vez?

File: packages/pilot/src/pages/PaymentLinks/PaymentLinks.js [hunk 3]
```diff
@@ -118,51 +165,86 @@ const PaymentLinks = ({
...
+    resetStepsRequest()
   }
 
+  const pagination = {
+    offset: filter.count * filter.page,
+    total: Math.ceil(totalPaymentLinks / filter.count),
+  }
+
+  const onExport = exportType => handleExport(
+    client.paymentLinks.exportData(filter, exportType),
```
Comment (eduardogspereira) [inline packages/pilot/src/pages/PaymentLinks/PaymentLinks.js:187]: Passar os dados

File: packages/cockpit/src/paymentLinks/search/query.test.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import buildQuery from './query'
+
+describe('PaymnetLinks Query', () => {
```
Comment (eduardogspereira) [inline packages/cockpit/src/paymentLinks/search/query.test.js:3]: Typo

File: packages/pilot/src/containers/PaymentLinks/index.js [hunk 5]
```diff
@@ -39,7 +52,38 @@ const PaymentLinks = ({
     />
     <Grid>
       <Row>
-        <NewLinksCard onAddPaymentLink={onAddPaymentLink} t={t} />
+        <Col
```
Comment (eduardogspereira) [inline packages/pilot/src/containers/PaymentLinks/index.js:55]: Da pra remover isso aqui? :thinking:

File: packages/pilot/src/pages/PaymentLinks/PaymentLinks.js [hunk 6]
```diff
@@ -76,17 +117,48 @@ const steps = {
...
   const [isNewLinkOpen, setIsNewLinkOpen] = useState(false)
+  const handleExport = useFileExporter({
+    exportHeaders,
+    exportParser,
+    exportPrefix,
+  })
+
+  useEffect(() => {
+    getLinksRequest(filter)
+  }, [getLinksRequest]) // eslint-disable-line react-hooks/exhaustive-deps
```
Comment (eduardogspereira) [inline packages/pilot/src/pages/PaymentLinks/PaymentLinks.js:144]: Remover este item

File: packages/pilot/src/pages/PaymentLinks/PaymentLinks.js [hunk 7]
```diff
@@ -76,17 +117,48 @@ const steps = {
...
+  paymentLinks,
+  previousStepRequest,
+  resetStepsRequest,
   step,
   t,
+  totalPaymentLinks,
 }) => {
   const [linkFormData, setLinkFormData] = useState(makeDefaulLinkData())
   const [isNewLinkOpen, setIsNewLinkOpen] = useState(false)
+  const handleExport = useFileExporter({
```
Comment (eduardogspereira) [inline packages/pilot/src/pages/PaymentLinks/PaymentLinks.js:136]: Não compensa deixar isso no proprio componente?

File: packages/pilot/src/pages/PaymentLinks/epic.js [hunk 8]
```diff
@@ -1,25 +1,62 @@
...
+  GET_LINKS_REQUEST,
+  RESET_FILTER_REQUEST,
+  resetFilterRequest,
 } from './actions'
 import paymentLinkSpec from '../../formatters/paymentLinkSpec'
 
+const resetLinksEpic = (action$, state$) => action$
+  .pipe(
+    ofType(RESET_FILTER_REQUEST),
+    delay(5000),
```
Comment (eduardogspereira) [inline packages/pilot/src/pages/PaymentLinks/epic.js:23]: Adicionar `// TODO: Precisamos adicionar uma melhor paginacao/ordenacao na rota paymentLinks para conseguir buscar os dado sem ser no ES`

**Pull request merged**